samples/bluetooth/mesh/shell_client/store_scenes.py

```diff
@@ -0,0 +1,22 @@
+""" This script sets up two scenes that are stored in the Scene Register. """
+
+from time import sleep
+import serial
+
+scene_no1 = 1            # Scene number to store
+scene_no2 = 2            # Scene number to store
+
+ser = serial.Serial()
+ser.baudrate = 115200
+ser.port = 'COM6'        # Serial port of client DK
```

Comment:
Maybe this can be done as a input from cmd line? The COM port will be different for each induvidual setup.
Applies to all your scripts.
https://www.w3schools.com/python/ref_func_input.asp